OWNERS

```diff
@@ -1,17 +1,18 @@
 reviewers:
-- skeeey
-- zhiweiyin318
 - qiujian16
+- skeeey
 - skitt
+- tpantelis
+- zhiweiyin318
```

Comment:
All approvers first (rearranged alphabetically)

Reply:
I'd think the whole list would be arranged alphabetically rather than two sub-lists arranged separately.